payloads/v1/webhooks.ts

```diff
@@ -15,6 +10,14 @@ export interface WebhookBadge {
     count?: number;
 }
 
+export interface WebhookBaseUser {
+    user_id?: number;
```

Comment:
Is it right that they are all optional? Or was this just a default?

Reply:
That's how it was in `WebhookUser`. I doubt that it's optional but this part was just a cut and paste.